src/linkmod.rs

```diff
@@ -175,24 +173,20 @@ fn make_request(request: &String) -> JsonValue {
 /// Makes an embed of search results. Takes a json array, and returns true {{{1
 /// if able to make an embed of all the results
 fn make_search_results_embed(message: &Message, results: JsonValue) -> bool {
-    let result = message.channel_id.send_message(|a| {
+    message.channel_id.send_message(|a| {
         a.embed(|b| {
             b.title("Search results:")
                 .description(&serialize_search_results(&results))
                 .color(Colour::from_rgb(255, 34, 108))
                 .timestamp(message.timestamp.to_rfc3339())
         })
-    });
-    if let Err(_) = result { false } else { true }
+    }).is_ok()
 }
 
 /// Takes a json array as input, returns a string of the search results serialized. {{{1
 fn serialize_search_results(results: &JsonValue) -> String {
     let mut final_string = String::new();
-    for (counter, entry) in results.members().enumerate() {
-        if counter > 10 {
-            break; //Don't put more than ten entries
-        }
+    for entry in results.members().take(10) {
```

Comment:
Eliminates the need for enumerating since the value was only used for the break.